### ISSUE TITLE
Update to not pull in the Supervisor cookbook

### ISSUE THREAD

**Issue by pvandervelde** [adamkrone/chef-consul-template#70]
The Supervisor cookbook depends on the deprecated Python cookbook so indirectly the consul-template cookbook depends on the Python cookbook but I'm pretty sure it doesn't need python.

**Comment by eyalzek** [comment]
This is the only thing holding me from getting rid of the `python` cookbook once and for all. A big +1

**Comment by dmigous** [comment]
+1

and it seems doesn't work with new Chef

```
================================================================================
Recipe Compile Error in /chef/local-mode-cache/cache/cookbooks/python/attributes/default.rb
================================================================================

NoMethodError
-------------
undefined method `python' for #<Chef::Node::Attribute:0x000000020be490>

Cookbook Trace:
---------------
  /chef/local-mode-cache/cache/cookbooks/python/attributes/default.rb:23:in `from_file'

Relevant File Content:
----------------------
/chef/local-mode-cache/cache/cookbooks/python/attributes/default.rb:

 16:  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 17:  # See the License for the specific language governing permissions and
 18:  # limitations under the License.
 19:  #
 20:  
 21:  default['python']['install_method'] = 'package'
 22:  
 23>> if python['install_method'] == 'package'
 24:    case platform
 25:    when "smartos"
 26:      default['python']['prefix_dir']         = '/opt/local'
 27:    else
 28:      default['python']['prefix_dir']         = '/usr'
 29:    end
 30:  else
 31:    default['python']['prefix_dir']         = '/usr/local'
 32:  end

System Info:
------------
chef_version=13.0.118
platform=debian
platform_version=8.7
ruby=ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
program_name=chef-solo worker: ppid=6;start=15:20:28;
executable=/opt/chef/bin/chef-solo
```

**Comment by pvandervelde** [comment]
Yeah that's what I got as well. Hence why I opened the issue. Does it work with a slightly less new Chef?

**Comment by eyalzek** [comment]
No problem on 12.19.36 besides the deprecation messages..

**Comment by dmigous** [comment]
So it won't be fixed in coming updates? And I should pin chef version below 13 to use this cookbook, right?

**Comment by rennhak** [comment]
+1

**Comment by skamv** [comment]
+1

**Comment by eyalzek** [comment]
Just realized that `supervisor` was added in the latest release which is pretty weird

**Comment by adamkrone** [comment]
@pvandervelde I have just pushed some changes that removes supervisor. I still need to run through the full test suite, but I got 16.04 to converge on Chef 13 so looking good so far.

**Comment by pvandervelde** [comment]
@adamkrone Sweet. Will grab the release when it comes out. Thanks heaps for fixing this.

**Comment by adamkrone** [comment]
@pvandervelde just released v0.13.0 on the supermarket. Let me know if you run into any issues.
